src/coreclr/src/inc/clrprivbinderutil.h

```diff
@@ -25,9 +25,7 @@
 
 //=====================================================================================================================
 // Forward declarations
-interface ICLRPrivAssembly;
-typedef DPTR(ICLRPrivAssembly) PTR_ICLRPrivAssembly;
-typedef DPTR(ICLRPrivBinder) PTR_ICLRPrivBinder;
+//interface ICLRPrivAssembly;
```

Comment:
Did you intentionally leave this commented out?

Reply:
I should have removed that, I have missed it when looking through the code before commit.